src/subdomains/core/statistic/cfp.service.ts

```diff
@@ -134,19 +141,18 @@ export class CfpService implements OnModuleInit {
     try {
       // update settings
       this.settings = await this.settingService.getObj<CfpSettings>('cfp');
-
+      // update masternodes
+      this.allMasternodes = await this.callApi<any>(this.myDefichainUrl);
+      try {
+        this.lockMasternodes = await this.callApi<any>(this.lockUrl);
+        this.cakeMasternodes = await this.callApi<any>(this.cakeUrl);
+      } catch (e) {
+        console.error('Failed to get lock/cake masternodes', e);
+      }
```

Comment:
Does it make sense to have this try-catch? I think if one step fails, all should fail.